cli/server.go

```diff
@@ -844,6 +844,12 @@ func (r *RootCmd) Server(newAPI func(context.Context, *coderd.Options) (*coderd.
 				}
 				defer closeWorkspacesFunc()
 
+				closeAgentStatsFunc, err := prometheusmetrics.AgentStats(ctx, logger, options.PrometheusRegistry, options.Database, time.Now(), 0)
```

Comment:
I think this should be disableable via a flag. We add lots of stats here for every single agent, and cloud metrics providers often charge per metric/datapoint so the costs can rack up quickly.

Reply:
Flag added